pkg/image/prune/prune.go

```diff
@@ -891,7 +889,7 @@ func (p *pruner) Prune(
 	glog.V(1).Infof("Using registry: %s", registryURL)
 
 	if err := p.registryPinger.ping(registryURL); err != nil {
-		return fmt.Errorf("error communicating with registry: %v", err)
+		return fmt.Errorf("error communicating with registry %s: %v", registryURL, err)
```

Comment:
I find these messages already too long and they are quite common. Also I will need to update documentation [here](https://github.com/openshift/openshift-docs/pull/4471/files#diff-ef0a42ebdb6104b972179790c68c2623R417). Looking at the current logging, it seems like this statement will sometimes triple the url. Could you please either remove it from the pinger or from here to make sure it's logged at least once but no more than twice?